CSharp/BatchExplorer/Models/PropertyDisplayModel.cs

```diff
@@ -42,5 +44,30 @@ public CollectionPropertyModel(string propertyName, List<PropertyModel> items)
         {
             this.Items = items;
         }
+
+        internal static string GetItemDisplayPrefix(IEnumerable collection)
+        {
+            var elementType = ElementType(collection.GetType());
+            if (elementType == null)
+            {
+                return "item";
+            }
+            return elementType.Name;
+        }
+
+        private static Type ElementType(Type collectionType)
+        {
+            Debug.Assert(collectionType != null);
+            Debug.Assert(typeof(IEnumerable).IsAssignableFrom(collectionType));
+
+            var stis = collectionType.FindInterfaces((m, c) => m.IsConstructedGenericType && m.GetGenericTypeDefinition() == typeof(IEnumerable<>), null);
```

Comment:
minor: Two stylistic things.
1) what is stis?  Can we use a more descriptive variable name?
2) Generally we've shied away from var, especially in cases where it's not clear what the variable type actually is -- in this case it's clear if you examine the code below (clearly stis must be a collection of some sort) but my personal preference is to spell the type out here.  If you don't want to do that (var is a religion =P) at least clarify the variable name.